src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1342,6 +1354,12 @@ internal SolidColorBrush GetBorderColor()
 
             ResetColorGlyphs();
 
+
+            if (this.NodeModel.IsCustomFunction)
+            {
+                result = (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["NodeCustomColor"];
```

Comment:
can you use nameof?

Reply:
perhaps do some null checking?

Reply:
> can you use nameof?

Hi @mjkkirschner ! I don't know if I can here :/ This is a dictionary key, and I am using similar way of getting the value to this, for example: https://github.com/DynamoDS/Dynamo/blob/ea6380044dd71f54c63256a601aeebcf8ba60183/src/VisualizationTests/HelixWatch3DViewModelTests.cs#L618

Check for nulls - done!!